.github/workflows/codeql.yml

```diff
@@ -0,0 +1,66 @@
+name: "CodeQL"
+
+on:
+  schedule:
+    # At 05:30 on every day-of-week from Monday through Friday.
+    - cron: "30 5 * * 1-5"
+
+jobs:
+  analyze-java:
+    name: Analyze Java (Kotlin)
+    runs-on:
+      labels: ubuntu-latest-8-cores
```

Comment:
er custom runners enabled for repositoriet? Jeg bestilte kun for `saksbehandling`, ref https://nav-it.slack.com/archives/C60FFACN5/p1691740172467899?thread_ts=1666796307.951419&cid=C60FFACN5

Reply:
Det er det vel, ettersom jobben kjører :)

Reply:
Usikker, men denne jobben verkar å vera lett, og går uansett kun ein gong i døgnet, så den kan fint køyre på ein heilt vanleg runner. Oppdaterer.